Realm/ObjectStore/object_store.cpp

```diff
@@ -29,6 +29,34 @@
 using namespace realm;
 using namespace std;
 
+template <typename T>
+static inline
+T get_value(TableRef table, size_t row, size_t column);
```

Comment:
I don't see any reason for these to be templates rather than just overloaded functions.